config/jobs/kubernetes/sig-release/release-branch-jobs/1.23.yaml

```diff
@@ -731,7 +731,6 @@ presubmits:
     max_concurrency: 12
     name: pull-kubernetes-e2e-kops-aws
     optional: true
-    run_before_merge: false
```

Comment:
I'd vote for keeping it when not removing the feature completely. Otherwise we will lose the context that the feature even exist.

Reply:
As per #27741:

> (Note that this is from a feature request from one of our internal users, internal tracking number is 253059454)

Considering that the author is from Google, this might be used internally by Google.

After some more investigation, it seems that this property somehow got introduced with #28022 but I'm not really sure how. Running the jobs generation again removes this property, and for some reason, I still didn't find a way to keep it.

Reply:
Hm, that's odd. If the generation removes the field then we may remove it for sanity reasons :+1:

Reply:
Note that `omitempty` was added after the initial implementation, so anything serialized in the interim would have that field (unnecessarily) added.

We (OpenShift) had this problem with our generated job files, which resulted in https://github.com/kubernetes/test-infra/pull/28074.